### PR TITLE
Migrate visualization tooltips to primereact

### PR DESCRIPTION
**Pull request by rpiaggio**
This simplifies the code, with no need to compute SVG resizing and positioning, while allowing us to render tooltips in regular HTML.

https://github.com/gemini-hlsw/explore/assets/1895643/cecd1ce5-7acd-491c-8b5c-a334805889f2

### CODE REVIEW

File: explore/src/main/scala/explore/visualization/GuideStarTarget.scala [hunk 1]
```diff
@@ -4,127 +4,34 @@
...
-          val (guideSpeedIcon, speedText) = p.analysis match
-            case AgsAnalysis.Usable(_, _, guideSpeed, _, _)                    =>
-              (speedIcon(guideSpeed).some, guideSpeed.tag.toSentenceCase)
-            case AgsAnalysis.NotReachableAtPosition(_, _, Some(guideSpeed), _) =>
-              (speedIcon(guideSpeed).some, guideSpeed.tag.toSentenceCase)
-            case _                                                             =>
-              (none, "")
-
-          val (speedWidth, _) = textDomSize(speedText)
-          val scaleTr         = s"scale(${1 / pf}, ${1 / pf})"
```
Comment (cquiroz) [inline explore/src/main/scala/explore/visualization/GuideStarTarget.scala:91]: 😢 😭

Comment (rpiaggio) [reply explore/src/main/scala/explore/visualization/GuideStarTarget.scala:91]: Sorry for that

**Review by cquiroz — APPROVED** [review]
LGTM

**Comment by cquiroz** [comment]
I think this may fix
https://app.shortcut.com/lucuma/story/2828/aladin-sticky-tooltips

**Pull request merged**